### PR TITLE
solving bug #707

### PR DESCRIPTION
**Pull request by hack3r-0m**
## What does this PR do?

Fixes environmental variables to be able to use external SMTP

## Test Plan

no new tests needed

## Related PRs and Issues

#707 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

## Doubtful Regarding :

What should be the default value for 
```
_APP_SMTP_HOST=smtp.sendgrid.net
_APP_SMTP_PORT=465
_APP_SMTP_SECURE=ssl
_APP_SMTP_USERNAME=<YOUR_USER_NAME>
_APP_SMTP_PASSWORD=<YOUR_API_KEY>
```
can anything go wrong if one doesn't want to use some vars from .env and they are still in `docker-compose.yml` ?

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -19,8 +19,11 @@ _APP_INFLUXDB_HOST=influxdb
 _APP_INFLUXDB_PORT=8086
 _APP_STATSD_HOST=telegraf
 _APP_STATSD_PORT=8125
-_APP_SMTP_HOST=maildev
-_APP_SMTP_PORT=25
+_APP_SMTP_HOST=smtp.sendgrid.net
```
Comment (eldadfux) [inline .env:22]: Key the default `maildev` hostname and port 25. Try to pass empty values for the other vars, I'm not sure if ti will work but lets see if the tests pass.

Comment (hack3r-0m) [reply .env:22]: maybe we can declare var and keep empty after =, so it will be empty

**Comment by hack3r-0m** [comment]
sorry I forgot docs belong to a different issue, I will add  docs altogether in other PR

**Comment by eldadfux** [comment]
Great work!

**Pull request merged**